boot/bootutil/src/loader.c

```diff
@@ -838,6 +838,11 @@ boot_validate_slot(struct boot_loader_state *state, int slot,
         if (BOOT_CURR_IMG(state) == 1) {
             min_addr = PM_CPUNET_APP_ADDRESS;
             max_addr = PM_CPUNET_APP_ADDRESS + PM_CPUNET_APP_SIZE;
+#ifdef PM_S1_ADDRESS
+    } else if (BOOT_CURR_IMG(state) == 0) {
```

Comment:
Fix alignment of line

Reply:
Fixed seems that a tabexpand went wrong sorry.